_episodes/01-introduction-to-dev-tools.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: episode
+title: "Getting to know tools for developing code"
+teaching: 20
+exercises: 25
+questions:
+  - "What tools are used for developing code"
+  - "What is a code editor"
+  - "What is an IDE?"
+objectives:
+  - "Clarify differences between code editors and IDE's"
+  - "Basic concepts of code editors and IDEs are introduced"
+  - "Demonstrate how various tools can be used for developing code"
+keypoints:
+  - "Choose a tool that is suitable for your workflow"
+---
+
+## Questions
```

Comment:
consider adding the question: how do you rename a function or variable across the entire project?